js/app/Run.js

```diff
@@ -52,4 +52,24 @@ app.run(function ($document, $rootScope, $transitions, BoardService) {
         OC.filePath('deck', 'img', 'app-512.png')
     );
 
+    $('#app-navigation-toggle').off('click');
+    // App sidebar on mobile
+    var snapper = new Snap({
+        element: document.getElementById('app-content'),
+        disable: 'right',
+        maxPosition: 250,
+        minDragDistance: 100
+    });
+
+    $('#app-navigation-toggle').click(function(){
+        if($(window).width() > 768) {
```

Comment:
remove this `if`, since the toggle should always be visible - then you won't need `!important` in CSS